x-pack/plugins/reporting/server/browsers/chromium/driver_factory/index.ts

```diff
@@ -66,18 +67,7 @@ export class HeadlessChromiumDriverFactory {
       proxy: this.browserConfig.proxy,
     });
 
-    return puppeteerLaunch({
-      userDataDir: this.userDataDir,
-      executablePath: this.binaryPath,
-      ignoreHTTPSErrors: true,
-      args: chromiumArgs,
-    } as LaunchOptions).catch((error: Error) => {
-      logger.error(
-        `The Reporting plugin encountered issues launching Chromium in a self-test. You may have trouble generating reports.`
-      );
-      logger.error(error);
-      return null;
-    });
+    return browserTest(this.binaryPath, chromiumArgs, logger);
```

Comment:
Maybe we don't actually need the self-checks at at startup any more? Can our the current self-check validation functions move to being called from the new diagnostic API? Maybe that'll be a long term goal?

Reply:
I just noticed you have moved those to `routes/diagnose` :)